server/api/genres.js

```diff
@@ -5,6 +5,7 @@ const { Genre, Artwork } = require('../db')
 // GET /api/genres/
 router.get('/', async (req, res, next) => {
   try {
+    // TODO: shouldn't have to include Artwork
     const genres = await Genre.findAll({
       include: [Artwork],
```

Comment:
Is the issue here that you need to include the artwork in order to get a count for the drop down?

Reply:
Yep

Reply:
not 100% sure if this would work but one idea is to create a virtual field and use the countArtwork method that Sequelize creates so we could have a property on the instance itself that would be a count of its associated artworks.